problems/water/water.adoc

```diff
@@ -52,18 +52,23 @@ check50 cs50/2017/x/water
 ----
 
 == Staff Solution
-
+Not sure how your solution should behave?
+Run our example by calling '~cs50/pset1/water'. Assumed that the underlined text is what some user has typed.
 [source]
 ----
-~cs50/pset1/water
+$ [underline]#~cs50/pset1/water#
+Minutes: [underline]#10#
+Bottles: 120
```

Comment:
I'm not sure if this example is really necessary since the `tl;dr` section shows what running the program should look like.

Reply:
I wasn't sure what were the CS50 documentation styles, I was going for repetition because to reduce confusion for an introductory course. Also, this being the first major assignment.